lib/formatters.js

```diff
@@ -1,27 +1,50 @@
 'use strict';
 
 const Promise = require('bluebird');
+const { DataEntity } = require('@terascope/utils');
 
 // No parsing, leaving to reader or a downstream op.
-function pass(data) {
-    return data;
+function raw(data, logger, metadata, opConfig) {
+    return Promise.map(data, (record) => {
```

Comment:
There is no need to use `Promise.map` since this is not async.

```suggestion
    return data.map(data, (record) => {
```

Reply:
fixed